openwisp_controller/config/tests/test_selenium.py

```diff
@@ -130,6 +132,48 @@ def test_create_new_device(self):
             'The Device “11:22:33:44:55:66” was added successfully.',
         )
 
+    def test_device_preview_keyboard_shortcuts(self):
+        self._create_config(device=self._create_device(name='Test'))
+        self.login()
+        self.open(reverse('admin:config_device_changelist'))
+
+        # open first device in the list
+        self.web_driver.find_element(
+            by=By.CSS_SELECTOR, value='tbody tr:nth-child(1) th a'
+        ).click()
+        try:
+            WebDriverWait(self.web_driver, 2).until(
+                EC.presence_of_element_located((By.CSS_SELECTOR, '#content-main'))
+            )
+        except TimeoutException:
+            self.fail('Device detail page did not load in time')
+
+        with self.subTest('press ALT + P and expect overlay to be shown'):
+            actions = ActionChains(self.web_driver)
+            actions.key_down(Keys.ALT).send_keys('p').key_up(Keys.ALT).perform()
+            try:
+                WebDriverWait(self.web_driver, 2).until(
+                    lambda driver: driver.find_element(
+                        By.CSS_SELECTOR, '.djnjc-overlay:not(.loading)'
+                    ).value_of_css_property('display')
+                    != 'none'
+                )
```

Comment:
Let's use [`EC.visibility_of_element_located`](https://www.selenium.dev/selenium/docs/api/py/webdriver_support/selenium.webdriver.support.expected_conditions.html#selenium.webdriver.support.expected_conditions.visibility_of_element_located)

```suggestion
                WebDriverWait(self.web_driver, 2).until(
                    EC.visibility_of_element_located(
                        (By.CSS_SELECTOR, '.djnjc-overlay:not(.loading)')
                    )
                )
```